source/features/clean-conversation-filters.tsx

```diff
@@ -42,44 +43,33 @@ function getCount(element: HTMLElement): number {
 	return Number(element.textContent.trim());
 }
 
-async function hideMilestones(): Promise<void> {
-	const milestones = await elementReady('[data-selected-links^="repo_milestones"] .Counter');
-	if (getCount(milestones!) === 0) {
-		(await elementReady('[data-hotkey="m"]'))!.parentElement!.remove();
+const MILESTONE_SELECTORS = ['#milestones-select-menu', '[data-testid="action-bar-item-milestones"]'];
+const PROJECT_SELECTORS = ['#project-select-menu', '[data-testid="action-bar-item-projects"]'];
+
+async function hide(container: HTMLElement): Promise<void> {
+	const milestones = $('[data-selected-links^="repo_milestones"] .Counter');
+	if (milestones && getCount(milestones) === 0) {
```

Comment:
Since the counter is missing, we should probably set up an API call like we do for the projects.

However since they have an overflow mechanism here I'm not sure it's worth it.

I'd leave a note with this information to also drop this in March 2025